k8s/src/main/scala/io/buoyant/k8s/istio/MixerApiRequests.scala

```diff
@@ -0,0 +1,96 @@
+package io.buoyant.k8s.istio
+
+import com.google.local.DurationProto.{Duration => GDuration}
+import com.fasterxml.jackson.databind.{DeserializationFeature, ObjectMapper}
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory
+import com.fasterxml.jackson.module.scala.DefaultScalaModule
+import com.fasterxml.jackson.module.scala.experimental.ScalaObjectMapper
+import com.twitter.util.Duration
+import istio.mixer.v1.{Attributes, ReportRequest, StringMap}
+
+import scala.io.Source
+
+object MixerApiRequests {
+
+  private[this] lazy val istioGlobalDict: Seq[String] = {
+    val yaml = Source.fromInputStream(
+      getClass.getClassLoader.getResourceAsStream("mixer/v1/global_dictionary.yaml")
+    ).mkString
+
+    val mapper = new ObjectMapper(new YAMLFactory) with ScalaObjectMapper
+    mapper.registerModule(DefaultScalaModule)
+    mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
+
+    mapper.readValue[Seq[String]](yaml).asInstanceOf[Seq[String]]
+  }
+
+  def mkReportRequest(
+    responseCode: Int,
+    requestPath: String,
+    targetService: String,
+    sourceLabelApp: String,
+    targetLabelApp: String,
+    targetLabelVersion: String,
+    duration: Duration
+  ): ReportRequest = {
+
+    val customWords = Seq(
+      "app",
+      "version",
+      requestPath,
+      targetService,
+
+      sourceLabelApp,
```

Comment:
@adleong In the original code, this would send the _values_ of these things as part of the dictionary. My understanding is that the dictionary should only contain the _keys_ to be resolved by Mixer. Is this intentional?